src/workflows/create.js

```diff
@@ -45,10 +45,11 @@ function updateUnitConfigs({ subworkflowData, unitConfigs }) {
  * @returns {*} subworkflow object
  */
 function createSubworkflowUnit({ appName, unitData, ...swArgs }) {
-    const { name: unitName, unitConfigs } = unitData;
+    const { name: unitName, unitConfigs, config } = unitData;
     const { subworkflows } = allWorkflowData;
     const { [appName]: dataByApp } = subworkflows;
     let { [unitName]: subworkflowData } = dataByApp;
+    subworkflowData.config = { ...subworkflowData.config, ...config };
```

Comment:
Maybe `Object.assign(subworkflowData.config, config)`?

Reply:
I would keep the spread notation. `subworkflowData.config` is not always defined, so `Object.assign()` will fail with an error, while `{ ...undefined }` does not.